### PR TITLE
chore(ui): add update member info admin button, add alert user has t…

### PR DESCRIPTION
**Pull request by LucasCharrier**
…o update email
Screenshot 2024-09-24 at 18 57 08
Screenshot 2024-09-24 at 19 15 45

### CODE REVIEW

File: src/components/MemberUpdate/MemberUpdate.tsx [hunk 1]
```diff
@@ -102,6 +102,12 @@ export const MemberUpdate = ({
                         title={alertMessage.title}
                     />
                 )}
+                <Alert
+                    className="fr-mb-8v"
+                    severity={"info"}
+                    closable={false}
+                    title={`Une fois la date de fin mise à jour, il faut que ${userInfos.fullname} réactive son email en rédéfinissant son mot de passe dans l'espace-membre`}
```
Comment (astranchet) [inline src/components/MemberUpdate/MemberUpdate.tsx:109]:
```suggestion
                    title={`Après la mise à jour de sa date de fin de mission, ${userInfos.fullname} doit redéfinir le mot de passe de son email dans son espace-membre pour le réactiver.`}
```

**Review by astranchet — APPROVED** [review]

File: src/components/MemberPage/MemberPage.tsx [hunk 2]
```diff
@@ -451,6 +451,25 @@ MemberPageProps) {
...
+                                <p
+                                    className="fr-text--xs"
+                                    style={{ fontStyle: "italic" }}
+                                >
+                                    Une information n'est pas à jour ?
+                                </p>
+
+                                <Button
+                                    linkProps={{
+                                        href: "/account/base-info",
```
Comment (revolunet) [inline src/components/MemberPage/MemberPage.tsx:465]: c'est la bonne url?

Comment (LucasCharrier) [reply src/components/MemberPage/MemberPage.tsx:465]: ah non en effet

Comment (LucasCharrier) [inline src/components/MemberPage/MemberPage.tsx:465]:
```suggestion
                                        href: `/community/${userInfos.fullname}/admin-update`,
```

**Pull request merged**